src/modules/koExtenders.ts

```diff
@@ -69,32 +69,53 @@ ko.bindingHandlers.contentEditable = {
 //TODO: Replace with logic that maintains a singular PlayerSpriteSVG rather than clone re-rendering
 ko.bindingHandlers.playerSpriteMove = {
     init: function (element) {
-        var observer = new IntersectionObserver(function (entries) {
-            entries.forEach(function (entry) {
-                if (entry.isIntersecting) {
-                    if (element.classList.contains('iconLocation')) {
-                        var targetElement = document.getElementById('playerSprite');
-                        var imageElement = element.cloneNode(true);
-                        imageElement.classList.remove('hide');
-
-                        targetElement.innerHTML = '';
-                        targetElement.appendChild(imageElement);
-
-                        //Get required variables to replicate the rotate on parent group as well
-                        var rotate = imageElement.getAttribute('rotate');
-                        var x = imageElement.getAttribute('localx');
-                        var y = imageElement.getAttribute('localy');
-
-                        if (rotate) {
-                            targetElement.setAttribute('transform', 'rotate(90,' + x + ', ' + y + ')');
-                        } else {
-                            targetElement.setAttribute('transform', '');
-                        }
+        function handleVisibleElement() {
```

Comment:
I can't tell if I'm just being picky, but something about declaring a function inside another function like just feels wrong. Weird how I didn't have as much of a problem when it was in the forEach...

What do you think? Can we declare this function outside of the binding handler, and pass in the element to act on? The way it is now, I'm pretty sure it creates a copy of this function for every playerSpriteMove binding we use (which is... 40-50 per region?)

Reply:
No, don't think it's picky at all. Honestly glad you pointed this out, as i'm not sure what possessed me to move it within the binding handler in the first place.... I first had it set-up like you suggested and it worked fine.

I think what sketched me out was the thought that the function felt out of place in a file full of bindingHandlers... 

Fixed. Clarified the TODO comments while i'm at it. Not sure if it's even worth getting round to that TODO but that'll be for another developer to consider, just pointing them in the right direction and giving the purpose.